### PR TITLE
Metrics Glossary: F1

### PR DESCRIPTION
**Pull request by y27choi**
### Linked issue(s):
[KOL-2579](https://linear.app/kolena/issue/KOL-2579/port-f1-score-doc-to-docskolenaio)

### What change does this PR introduce and why?
Adds F1 score doc

### Please check if the PR fulfills these requirements

- [ ] Include reference to internal ticket and/or GitHub issue "Fixes #NNNN" (if applicable)
- [ ] Relevant tests for the changes have been added
- [ ] Relevant docs have been added / updated

### CODE REVIEW

**Review by gordonhart — APPROVED** [review]

**Comment by gordonhart** [comment]
@y27choi ready to take another look. Should be good to merge

File: mkdocs.yml [hunk 1]
```diff
@@ -49,14 +49,18 @@ nav:
...
+  - Metrics Glossary:
+      - Metrics Glossary: metrics/index.md
+      - metrics/accuracy.md
+      - metrics/averaging-methods.md
+      - metrics/geometry-matching.md
+      - metrics/f1-score.md
+      - metrics/iou.md
+      - metrics/precision.md
+      - metrics/recall.md
+      - metrics/tp-fp-fn-tn.md
```
Comment (gordonhart) [inline mkdocs.yml]: Let me know if you want me to revert these changes if you'd prefer to land them on #105

**Comment by y27choi** [comment]
I like the threshold-dependent section and how you moved the f-beta section out from Limitations and Biases. Thanks for making these changes. 👍  Approved!
